includes/class-sensei-lesson.php

```diff
@@ -3822,6 +3822,53 @@ public function get_course_id( $lesson_id ) {
 
 	}
 
+	/**
+	 * Returns the course ID for a given set of lessons (with the key being the lesson ID)
+	 *
+	 * @since $$next-version$$
+	 * @access public
+	 *
+	 * @param array<int> $lesson_ids Array of Lesson IDs.
+	 * @return array<int, int|false> Array where the key is the lesson ID, and the value is the course ID (or false if not found)
+	 */
+	public function get_course_ids( $lesson_ids ) {
+		global $wpdb;
+
+		sort( $lesson_ids, SORT_NUMERIC );
+
+		$cache_key     = 'lesson/get-course-ids/' . implode( ',', $lesson_ids );
+		$cache_group   = 'sensei';
+		$cached_result = wp_cache_get( $cache_key, $cache_group );
```

Comment:
I'm using `wp_cache_get` and `wp_cache_set` as suggested by PHPCS thanks to our calls to `$wpdb`.